feat: Document the "No coverage data was sent" error DOCS-645 (#2011)
codacy/docs: docs/coverage-reporter/troubleshooting-coverage-cli-issues.md

## MalformedInputException while parsing report

If you get a `java.nio.charset.MalformedInputException` when running the Codacy Coverage Reporter it means that the coverage report includes a character that's not encoded in UTF-8. The invalid character can belong to the file name of one of your source code files, or even a class or method name.

For maximum compatibility of your coverage reports with the Codacy Coverage Reporter, make sure that your coverage reports use UTF-8 encoding and that they only include UTF-8 characters.

## No coverage data was sent

You can get the `No coverage data was sent` error when running the Codacy Coverage Reporter for the following reasons:

-   The coverage report doesn't include any coverage data.
-   The coverage report only includes data for files that don't exist in the associated Git repository. In this case, you also get one or more warnings `File: [filename] will be discarded and will not be considered for coverage calculation`.

> Make sure that your coverage report is not empty and that it includes coverage data for files in the associated Git repository.

Author: ```suggestion
Make sure that your coverage report isn't empty and that it includes coverage data for files in the associated Git repository.
```
